reference/docs-conceptual/Crescendo/advanced/using-crescendo-cmdlets.md

```diff
@@ -0,0 +1,224 @@
+---
+description: This article provides examples for using the Crescendo cmdlets to create a configuration. This can be used as an alternate for manually created the JSON configuration file.
+ms.date: 05/20/2022
+title: Create a Crescendo configuration using the Crescendo cmdlets
+---
+# Create a Crescendo configuration using the Crescendo cmdlets
+
+The Crescendo module includes a set of cmdlets that create various Crescendo object types. You can
+use these cmdlets to create a Crescendo configuration without the need to manually edit a JSON file.
+
+During the design of Crescendo, these cmdlets were created as helper functions that were not
```

Comment:
it would be fairer to say that these cmdlets were created before the decision that crescendo module and cmdlet creation could be better served by a declarative approach. Their utility was still obvious, so the decision was made to support both approaches.

Reply:
Updated the text.